src/test/java/com/distillery/intentsender/domain/command/CommandParamsValidatorTest.kt

```diff
@@ -10,18 +10,6 @@ class CommandParamsValidatorTest {
 
     private val validator = CommandParamsValidator()
 
-    @Test
-    fun `error returned when application id is set and component is not set`() {
```

Comment:
Should there be a test now verifying the component is added automatically?

Reply:
Nope, a component should only be added when it is set by a user. This test was ensuring that application id cannot be used without a component, which is not a case anymore.

But the class is not covered with the tests well enough :( So, ideally there should be one to verify intent params are properly added, but I simply do not have time to cover existing functionality with the test. Just trying to add some whenever I have a chance.